universal-application-tool-0.0.1/app/services/applicant/Block.java

```diff
@@ -219,6 +219,6 @@ public int hashCode() {
 
   @Override
   public String toString() {
-    return "Block [id: " + this.id + "]";
+    return "Screen [id: " + this.id + "]";
```

Comment:
Suggestion (blocking): This could potentially be confusing to a developer. Could you add a comment in this class saying `Screen` is the user-facing equivalent of `Block`?

Reply:
👍